widgets/headline/headline.php

```diff
@@ -31,6 +31,17 @@ function initialize(){
 		add_filter( 'siteorigin_widgets_wrapper_data_' . $this->id_base, array( $this, 'wrapper_data_filter' ), 10, 2 );
 	}
 
+	function get_settings_form() {
+		return array(
+			'responsive_breakpoint' => array(
+				'type' => 'measurement',
+				'label' => __( 'Responsive Breakpoint', 'so-widgets-bundle' ),
+				'default' => '780px',
+				'description' => __( 'This setting controls when the Mobile alignment settings will be used. The default value is 780px.', 'so-widgets-bundle' ),
```

Comment:
I know this is the standard description for this setting but it isn't the best. Let's perhaps change this to:

`The pixel resolution when the mobile alignment settings will be used.`

Reply:
Sounds good. Do you want to retain the "The default value is 780px" text?

Reply:
Super, thanks. I'd like to remove the `The default value is 780px.` text.